src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftAstToSql.kt

```diff
@@ -177,17 +177,32 @@ public open class RedshiftAstToSql(context: ScribeContext) : AstToSql(context) {
 
     /**
      * Type mappings for Redshift
-     * Shorten the naming
-     * - TIME WITH TIME ZONE -> TIMETZ
-     * - TIMESTAMP WITH TIME ZONE -> TIMESTAMPTZ
+     * - STRING -> VARCHAR(65535)
+     * - TINYINT -> error
      */
     override fun visitDataType(
         node: DataType,
         tail: SqlBlock,
     ): SqlBlock {
         return when (node.code()) {
-            DataType.TIME_WITH_TIME_ZONE -> tail concat type("TIMETZ", node.precision, gap = true)
-            DataType.TIMESTAMP_WITH_TIME_ZONE -> tail concat type("TIMESTAMPTZ", node.precision, gap = true)
```

Comment:
self-review: it seems like `TIMETZ` and `TIMESTAMPTZ` do not allow for specifying a precision parameter in Redshift. Changed to `TIME WITH TIME ZONE` and `TIMESTAMP WITH TIME ZONE` respectively, which works in Redshift and allows for a precision parameter